### ISSUE TITLE
NOT cli related issue but wayland gui blur related issue

### ISSUE THREAD

**Issue by Greatz08** [lmstudio-ai/lmstudio-bug-tracker#690]
Couldnt find any proper place so creating issue here. I am running arch linux with hyprland which runs on wayland. In that the issue i am facing while running the LM Studio is of blur. Text shown is blured and yea i know in wayland initial year there were scaling issues because of which people were facing blur issues in almost each electron based app and in some gtk and qt based apps too but right now its in pretty good state and i can run every app be it chromium , vs code or element or discord or firefox etc without any blur issues which were present in past and some still face today because of not having proper additional settings but i have so it works fine BUT LM STUDIO IS SHOWING BLURED TEXT idk why so can you guide me what else i can do to fix it.
Thankyou

**Comment by MarkoSchuetz** [comment]
On other electron apps I still can fix this by adding 

`--enable-features=UseOzonePlatform --ozone-platform=wayland`

to the command line, but I do not know how to do this with an appimage.

**Comment by MarkoSchuetz** [comment]
Turns out these arguments can just be added to the command line that starts the appimage:

`./LM-Studio-0.3.16-8-x64.AppImage --enable-features=UseOzonePlatform --ozone-platform=wayland`

**Comment by Greatz08** [comment]
yeah these arguments do work, it launches in xwayland session i guess, but i think native support for wayland will be best as all are progressing towards wayland :-))
